dashboards/component/stat/serializers.py

```diff
@@ -110,6 +119,9 @@ def get_value(self) -> Any:
     @classmethod
     def serialize(cls, **kwargs) -> StatSerializerData:
         self = cls()
+        filters = {}  # You need to obtain filters from the FilterComponent
```

Comment:
filter component object should be passed in to serialize instead.  you would then need to do something like 

```
f = ProductFilter(request.GET, queryset=self.get_queryset())
queryset = f.get_queryset()
```

This is not right, you will been to read the django filter docs to find out how to do it correctly